### PR TITLE
[IMP] travis: Add PyPi deploy

### PR DESCRIPTION
**Pull request by lasley**
Adds compatibility for auto-deploy as introduced in LasLabs/python-quality-tools#3

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,30 +1,40 @@
...
+    - VERSION="0.1.0"
+    - RELEASE="0.0.2"
+    - PROJECT="Python Red October"
+    - BRANCH_PROD="master"
+    - BRANCH_DOC="gh-pages"
+    - TESTS="0"
+    - LINT_CHECK="0"
+    - DOCS="0"
+    - PYPI="0"
+    - secure: "m5bpPi2ZU6vNchdCjGrj75NR2AWeGHTdPzpU7GIkintcaNA/wX9HPly7NsC91bZ0M3gzuY+xcJuUhKCTlHmDTnr2owP9bS21UFl6sd+WX1SfWcJibrcBdYVtqi3iJphoRnGFcPVq6KAx2s3IcitZivE++PdyLNac1gQgOzcshYcFEenVT+Q1kemPf8uC7bYZHrm75o6ukoeJXA1ydMxBk479YLtWkx6GefMJdCE4vN73SKUakuwZSgBD6s3EqntBWtf6jvFrIKnq0taGe8bSu0GMgX2BwBRrDkLzk0bKRH8B4cmb1MJ/0wCHboKculPVNNrqsYJ7tPlExUnLloPUUKXQKQHNJkaNV77fI8OrRyqu/jZW2IMsN/VgPHrzofmewM++Zd8IRugBjAJAC5BPl2nZQNn4QGoS7rDCjhG6phL+0YyM4wCdzcLBPrqJ4p1hDE6s2Hw6pSmDRie8lKHVMmxasbVUi/SP4lSUMzE/0UeRbTlWAoSA4ogIhWhOUy9P46tfzm2PhRjckx2MCH9Z4fPTWA44lK1SjwPRxBjC4o5Cj6fa4B5PRMSz+DaXyuB0uW0OB7pN9aLozkv7gKemv4GEeid8IWyHl7RwRv/OsCdLJ26+mJ/vqpei33ZxzhGHKjFxechCLa5n3SJm8jGAZicWv3xbiWyxR2SHsJH7PIw="
```
Comment (tedsalmon) [inline .travis.yml:25]: I'm not certain in what context this base64 is used but I'm assuming having it in the repo isn't a security risk? Just going off the name :)

**Review by tedsalmon — APPROVED** [review]
👍  LGTM minus a question I had about a variable.

File: .travis.yml (hunk 1 above)
Comment (lasley) [reply .travis.yml:25]: It's the PyPi password, encrypted per Travis docs - https://docs.travis-ci.com/user/encryption-keys/

I put a comment in the sample file showing how to generate - https://github.com/LasLabs/python-quality-tools/blob/master/sample_files/.travis.yml#L25

The `travis` binary is a Ruby gem, so it's just `gem install travis`

**Comment by lasley** [comment]
Don't merge this - I need a test branch

**Pull request merged**